source/backups.rst

```diff
@@ -271,7 +315,73 @@ Following steps are needed to restore a previously saved backup:
                endpointURL: https://URL-OF-THE-S3-COMPATIBLE-STORAGE
            ...
 
-   After that, the actual restoration process can be started as follows:
+2. After that, the actual restoration process can be started as follows:
+
+   .. code:: bash
+
+      kubectl apply -f deploy/backup/restore.yaml
+
+.. note:: Storing backup settings in a separate file can be replaced by passing
+   its content to the ``kubectl apply`` command as follows:
+
+   .. code:: bash
+
+      cat <<EOF | kubectl apply -f-
+      apiVersion: "pxc.percona.com/v1"
+      kind: "PerconaXtraDBClusterRestore"
+      metadata:
+        name: "restore1"
+      spec:
+        pxcCluster: "cluster1"
+        backupName: "backup1"
+      EOF
+
+.. _backups-pitr-restore:
+
+Restoring backup with point-in-time recovery
+********************************************
+
+.. note:: Disable the point-in-time functionality on the existing cluster before
+          restoring a backup on it, regardless of whether the backup was made
+          with point-in-time recovery or without it.
+
+If the point-in-time recovery feature :ref:`was enabled<backups-pitr-binlog>`,
+you can put additional restoration parameters to the ``restore.yaml`` file
+``pitr`` section for the most fine-grained restoration.
+
+* ``backupSource`` key should contain ``destination`` key equal to the s3 bucket
```

Comment:
I'm not sure about the `destination` key under `pitr/backupSource` - I don't think it is used. Mandatory is to have `s3` keys including the `bucket` key. We have removed destination in the example so probably it needs to be removed here also, but do check with @nonemax 
My understanding is that `destination` works for the full backup part of restore yaml, but not for pitr part.